### PR TITLE
Add daemon start/stop scripts

### PR DESCRIPTION
**Pull request by RikusW**
These are for convenience, I wrote them at work and it might be useful to others as well.

### CODE REVIEW

**Comment by andrew-elder** [comment]
This looks to be ok to me. Anyone else?

File: run_daemons.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Start all daemons
+
+nic=$1
```
Comment (davidcemin) [inline run_daemons.sh]: since it is accepting parameters, it would be good to have a -h option that would throw the usage.

**Comment by davidcemin** [comment]
Looks good to me

File: run_daemons.sh (hunk 1 above)
Comment (RikusW) [inline run_daemons.sh]: Updated for -h

File: run_daemons.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+if [ "$1" == "-h" ]; then
+        echo "Usage: $0 <network interface>"
+        echo "   eg: $0 eth1"
+        exit
+fi
+
+if [ "$1" == "" ]; then
+        nic="eth1" #edit as required
```
Comment (pinealservo) [inline run_daemons.sh]: Note for future improvement: It is helpful to have script parameters like this in variables that are defined at the start of the file, which makes it obvious what parameters can be edited and puts them all together.

File: stop_daemons.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Stop all daemons
+
+sudo killall maap_daemon
```
Comment (pinealservo) [inline stop_daemons.sh]: For possible future improvement:  Although killall is very convenient, it's not always provided in minimal OS configurations. I don't think there's a clear "best way" to do this across all bash-supporting systems, but there are probably a couple of ways that could be found in some cross-platform start/stop script implementations around.

**Comment by pinealservo** [comment]
I left a couple of comments for possible future improvement, but as they're convenience scripts that aren't required, I'm going to go ahead and merge since @andrew-elder and @davidcemin have already given their approval.

**Pull request merged**